src/app/forms/inputs/radio.component.js

```diff
@@ -8,6 +8,10 @@ export default function Radio(props) {
   const scope = useCss(css);
   const dataContext = React.useContext(DataContainerContext);
 
+  useEffect(() => {
+    dataContext.setData(props.dataKey, value);
```

Comment:
I think that calling `dataContext.setData` triggers a save to firebase, which is why we were only doing it on blur previously. Calling firebase after every letter they type might be too much. When the user is not logged in, `setData` just calls setState and is totally fine, though.

Was there a bug associated with calling it on blur? If not, let's change it back to use blur.

Reply:
I don't know if this is a bug, but it definitely is not user friendly. Trying so show a input box based off this value. The input box won't show up until you focus on the next input box.

Reply:
can you queue into the answer from the previous question to conditionally show the next field? like: https://github.com/JustUtahCoders/utahexpungements.org/pull/40/files#diff-31e7e0f2491482b249912bec08d7368fR128

Reply:
ah - i see that wouldn't work in this case because it is a text input, not a select

Reply:
With radio inputs, the value doesn't change nearly as often as text inputs. So I think it's fine to save to db every time the radio changes.

No need to change this.

Reply:
ignore all of my comments i was confused.